drivers/python/rethinkdb/_import.py

```diff
@@ -515,9 +523,9 @@ def csv_reader(task_queue, filename, db, table, options, progress_info, exit_eve
 
 # This function is called through rdb_call_wrapper, which will reattempt if a connection
 # error occurs.  Progress will resume where it left off.
-def create_table(progress, conn, db, table, pkey, sindexes):
+def create_table(progress, conn, db, table, pkey, sindexes, shards, replicas):
     if table not in r.db(db).table_list().run(conn):
-        r.db(db).table_create(table, primary_key=pkey).run(conn)
+        r.db(db).table_create(table, primary_key=pkey, shards=shards, replicas=replicas).run(conn)
```

Comment:
Hmm, yea I was afraid that might happen, which is why I originally had the default set to `1`. I was hoping that passing `None` would just work the same as doing nothing so that it would work with #4608 as @danielmewes. Is there a better way without having to make different calls depending on the value of `shards/replicas`?